tests/workflows_test.yaml

```diff
@@ -297,11 +298,12 @@ Workflows:
         description: Annotation Stats for {id}
         name: Annotation statistics report
         suffix: _stats.tsv
-      # - output: contig_mapping
-      #   data_object_type: Contig Mapping File
-      #   description: Contig mappings file for {id}
-      #   name: Contig mappings between contigs and scaffolds
-      #   suffix: _contig_names_mapping.tsv
+      - output: contig_mapping
+        data_object_type: Contig Mapping File
+        description: Contig mappings file for {id}
+        name: Contig mappings between contigs and scaffolds
+        suffix: _contig_names_mapping.tsv
+        optional: True
       - output: imgap_version
```

Comment:
Also add the assembly file from *_contigs.fna file with a data_object_type of 'Assembly Contigs' as an output file generated by [MetagenomeAnnotationActivity](https://microbiomedata.github.io/nmdc-schema/MetagenomeAnnotationActivity/)

Reply:
I'm going to do a second PR that will update the config.  So this will just be focused on the code changes.